doc/lua_api.md

```diff
@@ -8184,6 +8184,9 @@ child will follow movement and rotation of that bone.
 * `set_lighting(light_definition)`: sets lighting for the player
     * Passing no arguments resets lighting to its default values.
     * `light_definition` is a table with the following optional fields:
+      * `ambient_light` is a table controlling amount and color of ambient light (global lighting)
+        * `luminance` sets the amount of ambient light in range (0... LIGHT_SUN) like `light_source` has (default: `0`)
```

Comment:
It makes easier to adjust brightness of certain color. As the light level in MT ranges from 0 - 14, it is limited also to this range.

Reply:
I've sent you [a PR](https://github.com/Andrey2470T/minetest/pull/2) that turns this into a single color to demonstrate how this simplifies the code & API.

I don't think having a "redundant", discrete scalar with a few steps is a good idea. Modders can "scale" colors on their own to achieve the desired brightness. It would be rather confusing (to me at least) to have an additional "luminance" parameter with a limited amount of steps when the luminance is already encoded perfectly well in the color.